common/const.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 var (
-	RtyAttemNum   = uint(5)
-	RtyAttem      = retry.Attempts(RtyAttemNum)
-	RtyDelay      = retry.Delay(time.Millisecond * 400)
-	RtyErr        = retry.LastErrorOnly(true)
-	RetryInterval = 2 * time.Second
+	RtyAttemNum        = uint(5)
+	RtyAttem           = retry.Attempts(RtyAttemNum)
+	RtyDelay           = retry.Delay(time.Millisecond * 400)
+	RtyErr             = retry.LastErrorOnly(true)
+	RetryInterval      = 1 * time.Second
+	CacheClearInterval = 1000
```

Comment:
The name is not very clear - it seems to be the times of iteration.

Reply:
fixed